transport/rif_comms/node.py

```diff
@@ -177,7 +177,7 @@ def _run(self, *args: Any, **kwargs: Any) -> None:
             self.log.info("RIF Comms Node _run. Listening for messages.")
         except GreenletExit:  # killed without exception
             self.stop_event.set()
-            killall(self.greenlet)  # kill children
+            killall([self.greenlet])  # kill comms listener thread
```

Comment:
this was causing an error cause greenlet is not iterable and the method killall expects a list